.gitignore

```diff
@@ -64,3 +64,5 @@ local_settings.py
 *.db
 *.tar.gz
 tests/.env
+
+*\,covers
```

Comment:
what's this?

Reply:
They sprouts out with coverage annotation options. I played with it during my study on coverall. Including it in the gitignore will prevents some garbage for the future, nothing else